pkgs/applications/misc/quicksynergy/default.nix

```diff
@@ -0,0 +1,36 @@
+{stdenv, fetchurl, pkgconfig, gtk2, synergy}:
+let
+  version = "0.9.0";
+in
+stdenv.mkDerivation {
```

Comment:
change this to `stdenv.mkDerivation rec {` and move the version into the derivation